examples/flutter_firebase_login/lib/sign_up/cubit/sign_up_state.dart

```diff
@@ -4,24 +4,32 @@ class SignUpState extends Equatable {
   const SignUpState({
     this.email = const Email.pure(),
     this.password = const Password.pure(),
+    this.confirmPassword = const Password.pure(),
+    this.passwordMatch = FormzStatus.pure,
     this.status = FormzStatus.pure,
   });
 
   final Email email;
   final Password password;
+  final Password confirmPassword;
+  final FormzStatus passwordMatch;
   final FormzStatus status;
 
   @override
-  List<Object> get props => [email, password, status];
+  List<Object> get props => [email, password, status, confirmPassword, passwordMatch];
 
   SignUpState copyWith({
     Email email,
     Password password,
+    Password confirmPassword,
+    FormzStatus passwordMatch,
```

Comment:
I would recommend defining a ConfirmPassword `FormzInput` which has a dependency on the original password to perform the validation. See https://github.com/ChicagoFlutter/formz/issues/5 for more info.